src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userServ = new UserService();
```

Comment:
```suggestion
        UserService userService = new UserService();
```

should be class-level variable

Reply:
ok. Can I fix it here or create new commit and pull request?